py/sea/commands.py

```diff
@@ -770,7 +770,7 @@ def run (self, args, extra):
         self.harnessCmd = sea.ExtCmd (eharness)
         if args.cpu is None: args.cpu = 10
 
-        retval = self.harnessCmd.run (args, [eharness], encoding='utf-8', errors='replace', stdout=subprocess.PIPE)
+        retval = self.harnessCmd.run (args, [], encoding='utf-8', errors='replace', stdout=subprocess.PIPE)
```

Comment:
is this intentional? (`eharness` has been removed)

Reply:
yes, not sure why `eharness`(name of executable) was part of arguments in the first place; before removal, given an executable `debug.exe`, `ExecHarness` would call `debug.exe debug.exe`